website/content/en/docs/best-practices/observability-best-practices.md

```diff
@@ -0,0 +1,255 @@
+ 
+#### Metrics Naming
+Your operator metrics should align with the Kubernetes metrics names.
+
+Your operator users should get the same experience when searching for a metric across kubernetes operators, resources and custom resources.
+1. Check if a similar Kubernetes metric, for node, container or pod, exists and try to align to it.
+2. The metrics search list in the Prometheus and Grafana UI is sorted in alphabetical order. 
+When searching for a metric, it should be easy to identify metrics that are related to a specific operator.
+That is why we recommend that your operator metrics name to follow this format:
+`Operator name` prefix + the `Sub operator name` or `entity` + `metric name` based on the [Prometheus naming conventions](https://prometheus.io/docs/practices/naming/).
+
+**Note:** In Kubernetes metrics are separated like this:
+- node_network_**receive**_packets_total
+- node_network_**transmit**_packets_total
+In this example based on `receive` and `transmit`.
+
+Please follow the same principle and don't put similar metrics details as labels, so the user experience would be fluent.
+Example for this in an operator:
+- kubevirt_vmi_network_**receive**_errors_total
+- kubevirt_vmi_network_**transmit**_bytes_total
+- kubevirt_migrate_vmi_**data_processed**_bytes
+- kubevirt_migrate_vmi_**data_remaining**_bytes
+
+#### Recording Rules Naming
+As per [Prometheus](https://prometheus.io/docs/prometheus) documentation, [Recording rules](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/#recording-rules) allow you to precompute frequently needed or computationally expensive expressions and save their result as a new set of time series. 
+
+**Note:** The Prometheus recording rules appear in Prometheus UI as metrics.
+In order to easily identify your operator recording rules, their name should follow the same naming guidelines as the metrics.
+
+### Alerts Guidelines
+Clear and actionable alerts are a key component of a smooth operational
+experience. Ensuring we have clear and concise guidelines for developers and
+administrators creating new alerts will result in a better
+experience for end users.
+
+There should be a clear guidance aligning critical, warning, and info
+alert severities with expected outcomes across components, to avoide alerts fatigue for administrators.
+There must be an acceptance criteria for critical alerts.
+
+#### Recommended Reading
+
+A list of references on good alerting practices:
+
+* [Google SRE Book - Monitoring Distributed Systems](https://sre.google/sre-book/monitoring-distributed-systems/)
+* [Prometheus Alerting Documentation](https://prometheus.io/docs/practices/alerting/)
+* [Alerting for Distributed Systems](https://www.usenix.org/sites/default/files/conference/protected-files/srecon16europe_slides_rabenstein.pdf)
+
+#### Alert Ownership
+
+Individual teams are responsible for writing and maintaining alerting rules for
+their components, i.e. their operators and operands.
```

Comment:
Teh audience here is Operator Authors so I think we need re-wording a little

Reply:
Why. What I meant here is that each operator introduces and maintains their own alerts. How would you rephrase and is this section needed?

Reply:
I do think that we should use the word "teams" here. It would be "Operator Authors" as we can see in the other docs.

Reply:
Done